source/content.ts

```diff
@@ -121,6 +121,6 @@ import './features/hide-readme-header.css';
 import './features/hide-obvious-tooltips.css';
 import './features/clean-discussions.css';
 import './features/clean-rich-text-editor.css';
-
+import './features/show-asset-download-count.css';
```

Comment:
By the way only CSS-only features should be added here; JS features should import their own CSS file. Refer to my implementation of this issue